ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -467,6 +467,10 @@ spec:
           value: "$(params.pyxis_api_key_secret_name)"
         - name: pyxis_api_key_secret_key
           value: "$(params.pyxis_api_key_secret_key)"
+      workspaces:
+        - name: source
+          workspace: pipeline
+          subPath: src
```

Comment:
The hosted and release pipelines will need to pass the workspace like this as well. If you can please make the change I'll test it for you.

Reply:
Done